requirements.txt

```diff
@@ -18,6 +18,7 @@ Django<3.1
 html5lib>=1.0.1
 Pillow>=3.0
 six
-pytz
+pytz==2020.1
 gunicorn
-mysqlclient
\ No newline at end of file
+mysqlclient==2.0.1
+requests==2.24.0
```

Comment:
Do we want to use absolute versioning or minimal versioning? minimal might be better

Reply:
I have no preference and experience with either. Perhaps @Marti2203 can provide input?